app/javascript/components/editor/TestRunSummary.tsx

```diff
@@ -65,6 +65,18 @@ export const TestRunSummary = ({
   )
 }
 
+const TestRunSummaryStatus = ({
+  statusClass,
+  children,
+}: React.PropsWithChildren<{ statusClass: string }>): JSX.Element => {
+  return (
+    <div className={`summary-status ${statusClass}`} role="status">
+      <span className="--dot" />
+      {children}
+    </div>
+  )
+}
```

Comment:
This is a minor refactoring to extract some shared functionality into a component.